pkg/interactive-installer/README.md

```diff
@@ -0,0 +1,31 @@
+# EVE-OS interactive installer
+
+## Description
+
+The EVE-OS interactive installer provides a user-friendly text-based user interface (TUI) to configure the parameters of the installation. The primary goal of this installer mode is to facilitate the installation of EVE-OS on-edge devices while providing flexibility, and automation.
+The EVE-OS interactive installer provides a user-friendly environment for installing EVE-OS. Utilizing a TUI, allows users to configure installation parameters with ease. The primary goal of this installer is to simplify the installation process on edge devices while offering flexibility and automation, similar to the installation wizards used for setting up new Wi-Fi routers.
+
+## Configuration steps
+
+Users can configure the installation parameters through a series of steps, with the ability to navigate back and forth until their choices are finalized. The steps are as follows:
+
+1. Choose Filesystem Type: Select from options such as squashfs, EXT3, EXT4, and ZFS.
+2. Choose RAID Option: If ZFS is selected, choose whether to enable RAID.
+3. Choose Network Interface Card (NIC): Select the NIC to be used for networking.
+4. Choose Networking Configuration: Select either DHCP or static. For static configuration, input the IP address, subnet, gateway, and DNS servers.
```

Comment:
I thought you said you will defer this until a later phase when we can use the observatbility (and repair) tool.

Reply:
I forgot to update the README, sorry.